lib/extensions/extensions.js

```diff
@@ -6,14 +6,23 @@ const appConfig = require('../../app/config')
 // Utils
 const pathFromRoot = (...all) => path.join.apply(null, [__dirname, '..', '..'].concat(all))
 const pathToHookFile = packageName => pathFromRoot('node_modules', packageName, 'govuk-prototype-kit.config.json')
+const guarenteeArray = item => Array.isArray(item) ? item : [item]
```

Comment:
Hey Matt, sorry to be drive by nit picking but think you meant to say `guaranteeArray` maybe here.

Reply:
you can also avoid the `if` and just do this I think:

`const guaranteeArray = item => [].concat(item)`

probably a style choice, don't really mind either way